sdk/python/feast/infra/offline_stores/file_source.py

```diff
@@ -20,6 +20,7 @@
 class FileSource(DataSource):
     def __init__(
         self,
+        *,
```

Comment:
do we not need warnings in the other sources too?

Reply:
Other sources already have optional args(if they don't have positional args I didn't touch them.